runtime/interpreter/errors.go

```diff
@@ -718,7 +726,7 @@ func (e NonStorableValueError) Error() string {
 // NonStorableStaticTypeError
 //
 type NonStorableStaticTypeError struct {
```

Comment:
is this still a NonStorableStaticTypeError with the change?

Reply:
Yes, we are just using the sema type hierarchy now instead of the interpreter static type one (both represent static types).